index.html

```diff
@@ -3150,7 +3137,7 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
   <p class="changed">Unless the <a>processing mode</a> is set to `json-ld-1.0`,
     aliases of <a>keywords</a> are either <a data-lt="simple term definition">simple term definitions</a>,
     where the value is a <a>keyword</a>,
-    or a <a>expanded term definitions</a> with an `@id` <a>entry</a> and optionally an `@protected` <a>entry</a>;
+    or an <a>expanded term definitions</a> with an `@id` <a>entry</a> and optionally an `@protected` <a>entry</a>;
```

Comment:
```suggestion
    or an <a>expanded term definition</a> with an `@id` <a>entry</a> and optionally an `@protected` <a>entry</a>;
```

Reply:
Fixed in 7ba4fae23cef469dffc691d7f7c1c58b169ba905.